python/tests/test_tree_stats.py

```diff
@@ -67,22 +67,22 @@ def subset_combos(*args, p=0.5, min_tests=3):
     # of them, using this function, below. If we don't set a seed, a different
     # random set is run each time. Ensures that at least min_tests are run.
     # Uncomment this line to run all tests (takes about an hour):
-    # p = 1.0
+    p = 1.0
```

Comment:
TODO: revisit

Reply:
This appears to not be having the stated effect (of making tests take an hour); but I'd like to not address this here. I can confirm that tests are actually running.